pkg/chunk/chunk_store_test.go

```diff
@@ -602,8 +602,8 @@ func TestChunkStoreRandom(t *testing.T) {
 
 			// pick two random numbers and do a query
 			for i := 0; i < 100; i++ {
-				start := rand.Int63n(100 * chunkLen)
-				end := start + 1 + rand.Int63n((100*chunkLen)-start)
+				start := rand.Int63n(99 * chunkLen)
```

Comment:
why do we want this change?

Reply:
At the end of the list of chunks there are no more chunks.  It simplifies the test logic below to ensuring we never query over the end of the list of chunks.